src/RoboSaverVirtualModule.sol

```diff
@@ -222,7 +221,9 @@ contract RoboSaverVirtualModule {
             abi.encodeWithSelector(IVault.exitPool.selector, BPT_STEUR_EURE_POOL_ID, _card, payable(_card), request_);
         delayModule.execTransactionFromModule(address(BALANCER_VAULT), 0, payload, 0);
 
-        emit AdjustPoolTxDataQueued(address(BALANCER_VAULT), payload);
+        /// @dev event is leverage by off-chain service to execute the transaction
```

Comment:
i think it is better to make this part of the natspec where the event is defined

Reply:
2b2553a60543fa7c1fad034959f8f012a8243e5b